modules/persistent-storage-csi-vsphere-change-max-snapshot.adoc

```diff
@@ -0,0 +1,95 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-vsphere.adoc
+// * storage/persistent_storage/persistent-storage-csi-snapshots.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="vsphere-change-max-snapshot_{context}"]
+= Changing the maximum number of snapshots for vSphere
+
+The default maximum number of snapshots per volume in vSphere Container Storage Interface (CSI) is 3. You can change this maximum number of snapshots to a maximum of 32 per volume. 
+
+However, be aware that increasing the snapshot maximum involves a performance trade off, so for better performance use only 2 to 3 snapshots.
+
+.Prerequisites
+
+* Access to the cluster with administrator rights.
+
+.Procedure
+
+. Check the current config map by the running the following command:
++
+[source, terminal]
+----
+$ oc -n openshift-cluster-csi-drivers get cm/vsphere-csi-config -o yaml
+----
++
+.Example output
++
+[source, terminal]
+----
+apiVersion: v1
+data:
+  cloud.conf: |+
+    # Labels with topology values are added dynamically via operator
+    [Global]
+    cluster-id = rbednar-vsphere-01-cwv8p
+
+    [VirtualCenter "vcenter.devqe.ibmc.devcluster.openshift.com"]
+    insecure-flag           = true
+    datacenters             = DEVQEdatacenter
+    migration-datastore-url = ds:///vmfs/volumes/vsan:527320283a8c3163-2faa6dc5949a3a28/
+
```

Comment:
Do we want to add a `[Snapshot].global-max-snapshots-per-block-volume` to this example, or state that this config map uses the default, so that folks know why they are looking at the CM?

Reply:
I believe that this is a new field, so when changing the max for the 1st time, it won't be there.